app/assets/stylesheets/pageflow/editor/blank_entry.css.scss

```diff
@@ -1,13 +1,14 @@
 .blank_entry {
-  margin: 0 auto;
-  top: 5%;
+  margin: 0;
   position: relative;
   font-family: "SourceSansPro";
   font-size: 17px;
   letter-spacing: 0.01em;
   text-align: center;
-  width: 50%;
+  padding: 5% 25%;
+  background-color: #fff;
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.